errors/interface.go

```diff
@@ -83,3 +84,11 @@ func (err *wrapErrorTracer) Error() string {
 func (err *wrapErrorTracer) Trace() []string {
 	return err.trace
 }
+
+func Is(err, target error) bool {
```

Comment:
package name collision. This package supplants the built in. We wrote this back when there was no error traces on errors in the standard libraries. There actually is now, and we could switch to it, but it will require some refactoring in gadget and generate.

Reply:
'Is' and 'As' were added in 1.13